.env.ecs

```diff
@@ -0,0 +1,12 @@
+SOLR_CORE=blacklight-development
+S3_SOURCE_BUCKET_NAME=yale-image-samples
+S3CACHE_BUCKET_NAME=yale-image-samples/cantaloupe/cache
+PASSENGER_APP_ENV=development
+# Required by Postgres / Blacklight
+POSTGRES_HOST=localhost
+POSTGRES_DB=blacklight_yul_development
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=password
+# Point to an external manifests service
+IIIF_MANIFESTS_BASE_URL=/manifests/
+SOLR_URL=http://localhost:8983/solr/blacklight-development
```

Comment:
Is there a way to make the relationship between the `.env` file and the `.env.ecs` file clearer and less duplicative? I'm worried that they'll get out of sync in unhelpful ways.

I've separated the solr_url and solr_core in the management app, so I'll separate them here.

Reply:
Yes, we need to discuss and decide on conventions as a team about what goes where.  I think figuring out how to keep dev & prod as similar as possible with also help guide the solution.  In a future pass, I think we might be able to get away without needing .env.ecs.  For the first pass, I was trying to limit how much I had to change from what was in place already (I know, I didn't do a very good job - lots changed)